app/ts/components/subcomponents/PasteCatcher.tsx

```diff
@@ -7,20 +7,15 @@ interface PasteProps {
 
 // catches paste events if enabled == true and callbacks OnPaste
 export function PasteCatcher(props: PasteProps) {
-
 	function catcher(event: Event) {
 		if (props.enabled === false) return
 		if (!(event instanceof ClipboardEvent)) return
 		if (event.clipboardData === null) return
-		props.onPaste(event.clipboardData.getData('text'));
+		props.onPaste(event.clipboardData.getData('text'))
 	}
-
 	useEffect(() => {
 		window.addEventListener('paste', catcher)
-		return () => {
-			window.removeEventListener('paste', catcher);
-		}
+		return () => { window.removeEventListener('paste', catcher) }
 	}, [props.enabled, props.onPaste])
-
-	return <div></div>;
+	return <div></div>
```

Comment:
are the blank `div` pairs needed here?

```suggestion
	return <></>
```